person.rb

```diff
@@ -1,17 +1,16 @@
-require_relative 'sudent'
-require_relative 'teacher'
-require_relative '.corrector'
+require_relative 'corrector'
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1...100)
     @name = name
     @parent_permission = parent_permission
     @age = age
     @corrector = Corrector.new
+    @rentals = []
```

Comment:
- [ ] Kindly include a method for adding rentals and create a `has-many` association between the rental and person class.